todo_app/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'todos#index'
+  resources :todos do
+end
```

Comment:
If you're not going to put anything inside of the do-end block, then you can just write this out as `resources :todos`